pyproject.toml

```diff
@@ -49,19 +49,21 @@ artifacts = [
   "*.pyccel",
 ]
 include = [
-  "include",
+  "pyccel/extensions/STC/include",
   "*.py",
   "*.pyh",
   "*.pyccel",
   "pyccel/parser/grammar/*.tx",
   "pyccel/stdlib/*/*.h",
   "pyccel/stdlib/*/*.c",
   "pyccel/stdlib/*/*.f90",
-  "pyccel/extensions/STC/include/*/*.c"
+  "pyccel/extensions/STC/include/*/*.c",
+  "pyccel/extensions/gFTL/include/v2"
 ]
 exclude = [
     "pyccel/extensions/STC/src",
-    "pyccel/extensions/STC/misc"
+    "pyccel/extensions/STC/misc",
+    "pyccel/extensions/gFTL/tests"
```

Comment:
Do you know which `include` is causing the `tests/` folder to be included? I don't see any Python files

Reply:
I will remove this line in my next commit. 
The reason for excluding the `tests` folder is that it was added with `include`, but now that I have changed it to 'pyccel/extensions/STC/include', there is no need to exclude it.

Reply:
Fixed : a06f44e